plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -29,11 +35,11 @@ open class KtlintExtension {
     var ignoreFailures = false
 
     /**
-     * Report output format.
+     * Report output formats.
      *
      * Available values: plain, plain_group_by_file, checkstyle, json.
      *
-     * Default is plain.
+     * Default is empty.
      */
-    var reporter: ReporterType = ReporterType.PLAIN
```

Comment:
Do we want to deprecate this before outright removing it? @Tapchicoma thoughts?

Reply:
Yeah, I think deprecation will be better. Less painful to update the plugin.

@JLLeitschuh do you know if https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-deprecated/index.html will work here? Would be very useful to be able to update this field using IDE action.

Reply:
Probably would. If were were to hard code the refactor suggestion to setting the field to be `setOf(ReporterType.PLAIN)`.

Not 100% certain though.

Reply:
I did it as `setOf(ReporterType.PLAIN)`

Reply:
Our idea is to have deprecated `var reporter` field and new `var reporters` and add following annotation for deprecated one:
```
@Deprecated(
    message = "Ktlint introduced multioutput support since 0.11.1 version",
    replaceWith = ReplaceWith("reporters = setOf(x)", "reporter"))
```